package.json

```diff
@@ -82,6 +82,8 @@
     "typescript": "^2.5.2"
   },
   "dependencies": {
+    "@types/progress": "^2.0.1",
```

Comment:
@types should be a devDependency, not a dependency.

Reply:
Oh, good point.